device_modules/fdo_sys/fdo_sys.c

```diff
@@ -75,7 +75,7 @@ int fdo_sys(fdo_sdk_si_type type,
 			fdow = ModuleAlloc(sizeof(fdow_t));
 			if (!fdow_init(fdow) || !fdo_block_alloc_with_size(&fdow->b, MOD_MAX_BUFF_SIZE)) {
 #ifdef DEBUG_LOGS
-				printf(
+				LOG(LOG_DEBUG,
 					"Module fdo_sys - FDOW Initialization/Allocation failed!\n");
 #endif
```

Comment:
Any reason why are we still retaining teh '#endif' still ? I guess we can remove it.